libs/sdk-ui-kit/src/SettingWidget/StylingSettingWidget/StylingSettingWidget.tsx

```diff
@@ -147,14 +149,14 @@ const StylingSettingWidgetCore = <T extends StylingPickerItemContent>(
                     <FooterButtons>
                         <Button
                             className="gd-button-secondary"
-                            onClick={handleCancel}
-                            disabled={isApplyButtonDisabled}
+                            onClick={onCancel}
+                            disabled={disabledCancelButton ?? isApplyButtonDisabled}
```

Comment:
It should be:
`disabled={disabledCancelButton !== undefined ? disabledCancelButton : isApplyButtonDisabled }`